Backend/Backend/Infrastructure/AppDbContext.cs

```diff
@@ -146,7 +146,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             .HasMany(e => e.Ratings)
             .WithOne(e => e.Podcast)
             .HasForeignKey(e => e.PodcastId)
-            .IsRequired();
+            .OnDelete(DeleteBehavior.Restrict);
```

Comment:
Fairly certain you do not want this restricted. If you delete a podcast, the ratings should come with it

Reply:
Actually, there were two 1 to many relation on rating entity so it was causing error